source/contracts/reporting/ReputationToken.sol

```diff
@@ -108,6 +116,36 @@ contract ReputationToken is DelegationTarget, Extractable, ITyped, Initializable
         return universe;
     }
 
+    function getTotalMigrated() public view returns (uint256) {
+        return totalMigrated;
+    }
+
+    function updateSiblingMigrationTotal(IReputationToken _token) public returns (bool) {
+        require(_token != this);
+        IUniverse _supposedUniverse = _token.getUniverse();
+        require(_token == universe.getParentUniverse().getChildUniverse(_supposedUniverse.getParentPayoutDistributionHash()).getReputationToken());
+        totalTheoreticalSupply += migratedToSibling[_token];
+        migratedToSibling[_token] = _token.getTotalMigrated();
+        totalTheoreticalSupply -= migratedToSibling[_token];
+        return true;
+    }
+
+    function updateParentTotalTheoreticalSupply() public returns (bool) {
+        IUniverse _parentUniverse = universe.getParentUniverse();
+        totalTheoreticalSupply -= parentTotalTheoreticalSupply;
+        if (_parentUniverse == IUniverse(0)) {
+            parentTotalTheoreticalSupply = Reporting.getInitialREPSupply();
+        } else {
+            parentTotalTheoreticalSupply = _parentUniverse.getReputationToken().getTotalTheoreticalSupply();
+        }
+        totalTheoreticalSupply += parentTotalTheoreticalSupply;
```

Comment:
Still not following the subtraction then adding to totalTheoreticalSupply. This is called from the constructor, what is thought to be parentTotalTheoreticalSupply initially (0)?

Reply:
Explained the theoretical supply calculations above. Initially it is 0. Solidity defaults to all uninitialized values being 0 or for bools false.